tests/ui_tests/app/page/provider_list/page.py

```diff
@@ -96,3 +103,10 @@ def wait_state():
             assert state_after != self.table.LOADING_STATE_TEXT
 
         wait_until_step_succeeds(wait_state, period=1, timeout=self.default_loc_timeout)
+
+    @allure.step("Delete host")
+    def delete_provider_by_row(self, row: WebElement):
```

Comment:
Above you use `click_name_in_row`, here you use **by**. I think it's better to use "in"

Reply:
renamed